### DISCUSSION TITLE
i18next validation

### DISCUSSION THREAD

**Discussion by pradyuk** [twilio-labs/paste#2004] [General] [answered]
In console zen, I am trying to use [eslint-plugin-i18next](https://github.com/edvardchen/eslint-plugin-i18next) to catch all strings which are not wrapped in translate functions. However, the text component of paste does not use quotes on the strings, so the plugin does not catch them. Any ideas or recommendations on how to tackle this?

**Comment by TheSisb** [comment, marked as answer]
Hello @pradyuk,

According to [their docs](https://github.com/edvardchen/eslint-plugin-i18next#html-markup), it should translate just fine.  The Paste Text component is used like this:
```js
<Text>Hello</Text>
```
which matches what they claim to be detecting. Can you elaborate on what specific issue you're having?

**Reply by kylegilbreath** [reply]
Hi @pradyuk! Just following up on this. Could you provide some more detail on the issue you were experiencing?

**Reply by loreina** [reply]
Marking this as answer due to inactivity. If you need more support, please respond in this discussion or create a new one and our team will be happy to help!
